### PR TITLE
Remove explicit returns to simplify code

### PR DESCRIPTION
**Pull request by jevonmao**
Since Swift 5.1, [SE-0255](https://github.com/apple/swift-evolution/blob/master/proposals/0255-omit-return.md) evolved Swift to have the capability to implicitly return from functions.

I removed unnecessary explicit returns to improve code readability and simplicity.

### CODE REVIEW

**Comment by ldiqual** [comment]
I'm no longer very active in the swift world but whatever language we're working with, I think having `return` is more explicit and clearer than not having it. Ultimately it comes down to taste so I'll defer to @JannThomas and @SplittyDev, but what I'd like to push for is to add swiftlint to the repo and enforce one style or the other – so that there's no debate about it in future PRs.

**Comment by jevonmao** [comment]
@ldiqual I understand your perspective. Ultimately is your's and the maintainer's call. I am not sure if SwiftLint supports this particular `return` scenario, but I agree, that adding SwiftLint is a good idea.

**Comment by SplittyDev** [comment]
I'm personally a fan of not having explicit `return` statements. Let's wait for @JannThomas and see what he prefers.

**Comment by JannThomas** [comment]
To be honest, I don't have a strong personal opinion on this, but this is where the Swift community is going and I think we should probably adopt it if only for that reason.

**Review by SplittyDev — APPROVED** [review]
LGTM 🚀

**Comment by JannThomas** [comment]
That said lets also add SwiftLint, I created Issue #107 for that and will work on it I the coming days.

**Review by JannThomas — APPROVED** [review]

**Pull request merged**
